crates/agentgateway/proto/resource.proto

```diff
@@ -538,6 +539,23 @@ message PolicySpec {
     FailureMode failure_mode = 2;
   }
 
+  message MCPAuth {
+    string issuer = 1;
+    string audience = 2;
+    optional string jwks_url = 3; 
```

Comment:
i dont think we need optional keyword in proto v3?

Reply:
we're using it for other fields: https://github.com/agentgateway/agentgateway/blob/main/crates/agentgateway/proto/resource.proto#L138 But maybe we can just leave it off?